src/sentry/api/endpoints/accept_organization_invite.py

```diff
@@ -121,6 +131,13 @@ def post(self, request: Request, member_id: int, token: str) -> Response:
         else:
             response = Response(status=status.HTTP_204_NO_CONTENT)
 
+        organization_member = helper.om
+        organization = organization_member.organization
```

Comment:
Is the `organization` loaded with the member or will this cause another DB query?

Reply:
it's a field on the OrganizatiomMember model so i think it's already loaded

Reply:
Django won't eager load associations unless you tell it to when the original entity is loaded. Looking at how the InviteHelper works it isn't eagerloading the organization. It probably should be changed to eagerload as the `get()` method is also accessing member.organization as well.